### DISCUSSION TITLE
Drag and drop on nested table rows

### DISCUSSION THREAD

**Discussion by AshutoshSajan** [ant-design/ant-design#28501] [General]
can we use Drag and drop on nested table rows? if yes, then please help me with that. it will be very helpful.

**Comment by fernandaramos84** [comment]
We need a drag and drop list too... I found two of them but I don't know if it is safe to use:
https://motion.ant.design/exhibition/demo/list-sort    
https://codesandbox.io/s/50w9wz7j4x
Is there a way this can be added to the default code of ant if secure?
